docs/help/contributor/README.md

```diff
@@ -0,0 +1,6 @@
+# Knative Contributor's Guide (WIP)
+
+## Table of Contents
+
+- [Knative's MkDocs Conributor Guide](./mkdocs-contributor-guide.md)
+- [Knative's Style Guide](./style-guide/README.md)
```

Comment:
Small nit but I'd prefer we don't use possessive for inanimate objects, i.e. "Knative" not "Knative's"

Reply:
yeah no problem I think I actually messed up the wording here